src/components/TokenBalance.tsx

```diff
@@ -119,14 +131,62 @@ function useErrorMessageWithRefresh() {
   }, [shouldShowError])
 }
 
-export function AssetsTokenBalance() {
+export function AssetsTokenBalance({ showInfo }: { showInfo: boolean }) {
   const { t } = useTranslation()
 
+  const [infoVisible, setInfoVisible] = useState(false)
+  const [shouldRenderInfoComponent, setShouldRenderInfoComponent] = useState(false)
```

Comment:
it's important to stop rendering the tooltip (instead of leaving it there with opacity 0) because it sits on top of the segmented control and will prevent the segmented control from receiving the tap action